### PR TITLE
Footer responsive

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
fixed bug in cart and made ecom reponsive

### CODE REVIEW

File: src/css/cart.css [hunk 1]
```diff
@@ -138,15 +133,39 @@
...
     justify-content: space-between;
     align-items: center;
-}
\ No newline at end of file
+}
+
+
+.checkout{
+    position: sticky;
+    top: 120px;
```
Comment (AnjaliDeshwani) [inline src/css/cart.css:156]: Suggestion- Try to be consistent with units.

File: src/css/productListing.css [hunk 2]
```diff
@@ -69,7 +73,7 @@
 
 .product-items-container{
     width: 100%;
-    margin-left: 260px;
+    margin-left: 10px;
```
Comment (AnjaliDeshwani) [inline src/css/productListing.css:76]: Here also, as at some places you've used rem and at some px.

**Review by AnjaliDeshwani — APPROVED** [review]
Except the mentioned suggestion.
Rest things looks to go ahead.

File: src/css/cart.css (hunk 1 above)
Comment (mohammadkhizerkhan) [reply src/css/cart.css:156]: sure, thanks

**Review by JowelTisso — APPROVED** [review]
Looks Good,
Points: Unused codes are removed

File: src/css/cart.css [hunk 3]
```diff
@@ -138,15 +133,39 @@
     align-self: flex-start;
     border-radius: 4px;
     box-shadow: var(--shadow-cart);
+    position: sticky;
+    top: 120px;
```
Comment (manismk) [inline src/css/cart.css:137]: try to use relative unit

File: src/css/productListing.css [hunk 4]
```diff
@@ -1,17 +1,22 @@
...
+/* .hammerIcon{
+    display: none;
+} */
+
+
+
+.filter-container {
     color: var(--text-color);
-    width: 250px;
+    width: 305px;
```
Comment (manismk) [inline src/css/productListing.css:15]: try to use relative unit

File: src/css/productListing.css [hunk 5]
```diff
@@ -1,17 +1,22 @@
...
-
 .product-container{
     display: flex;
     margin: 7rem 0 1rem 0;
 }
 
-.filter-container{
+/* .hammerIcon{
+    display: none;
+} */
```
Comment (manismk) [inline src/css/productListing.css:9]: Remove the commented code

File: src/css/productListing.css [hunk 6]
```diff
@@ -107,7 +111,26 @@
...
     width: 100%;
     height: 16rem;
+    position: relative;
 }
+
+
+
+.offer-icon {
+    position: absolute;
+    color: #9ce7ff;
```
Comment (manismk) [inline src/css/productListing.css:121]: If possible, covert this to css variable

File: src/css/responsive.css [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+    }
+    .product-items{
+        width: 130vw;
+        grid-template-columns: repeat(auto-fill, minmax(190px, 210px));
+    }
+    .card-body{
+        height: 7rem;
+    }
+    .card-product{
+        height: 451px;
```
Comment (manismk) [inline src/css/responsive.css:97]: try to use relative unit

File: src/components/productpage/productlisting/ProductListing.jsx [hunk 8]
```diff
@@ -1,15 +1,16 @@
-import React from "react";
+import React, { useState } from "react";
 import { Category, FilterHeader, Range, Rating, SortBy } from "../filter";
 import { useData } from "../../../store/data/DataContext";
 import { useFilter } from "../../../store/data/FilterContext";
 import {Product} from "./Product.jsx"
+import { BsFilterLeft } from "react-icons/bs";
+// import {filterToggleHandler} from "../../../services/utils"
```
Comment (manismk) [inline src/components/productpage/productlisting/ProductListing.jsx:7]: Remove the commented code

File: src/components/cart/Cart.jsx [hunk 9]
```diff
@@ -123,12 +126,23 @@ function Cart() {
...
+                      >
+                        Move To Wishlist
+                      </button>
+                    )}
                     <button
                       class="btn product-btn btn-s"
-                      onClick={() =>
-                        removeCartHandler(cartItem)
-                      }
+                      onClick={() => removeCartHandler(cartItem)}
```
Comment (manismk) [inline src/components/cart/Cart.jsx:145]: change class to className

File: src/components/cart/Cart.jsx [hunk 10]
```diff
@@ -150,34 +164,41 @@ function Cart() {
...
             </div>
-            <div class="Delivery charges-amt">
-              <p>Delivery Charges</p>
-              <h3>&#8377;{totalSummary.totalDeliveryCharge}</h3>
+            <div className="checkout">
+              <p>
+                You will save &#8377; {totalSummary.totalSavedAmount} on this
+                order
+              </p>
+              <button class="btn cart-btn btn-s order-btn">PLACE ORDER</button>
```
Comment (manismk) [inline src/components/cart/Cart.jsx:198]: change class to className

**Review by manismk — APPROVED** [review]
Looks good.
Good naming convention

**Pull request merged**